### ISSUE TITLE
Drop in performance when a script is open in the editor

### ISSUE THREAD

**Issue by tadaaagames** [defold/editor2-issues#2855]
### Expected behaviour

No slowdowns when a script is in focus in the editor

### Actual behaviour

I have my macbook pro 13 hooked up to an external monitor (no dual screen used).

If I have a script or lua file in focus in the editor, the whole system (moving windows, navigating menu, running a build) becomes extremely sluggish.
As soon as the focus is moved to something else in the editor, like a collection, an atlas or a .MD file, everything goes back to being smooth and nice.

### Steps to reproduce

- Open a project in defold
- Open a .script or a .lua file and live it in focus (visible)
- Launch a build and then try to move a window around

<hr/>

<table>
<tr><td>Build time</td><td>2019-11-29T13:37:43.182609</td></tr>
<tr><td>Defold channel</td><td>editor-alpha</td></tr>
<tr><td>Defold editor sha</td><td>39b2b2d4bd91ffd166a5776bd998912a0ec9758c</td></tr>
<tr><td>Defold engine sha</td><td>2be2687cbb670c2dbe9cf2e99577bc3338561778</td></tr>
<tr><td>Defold version</td><td>1.2.164</td></tr>
<tr><td>GPU</td><td>Intel(R) Iris(TM) Plus Graphics 640</td></tr>
<tr><td>GPU Driver</td><td>2.1 INTEL-12.10.12</td></tr>
<tr><td>Java version</td><td>11.0.1+13</td></tr>
<tr><td>OS arch</td><td>x86_64</td></tr>
<tr><td>OS name</td><td>Mac OS X</td></tr>
<tr><td>OS version</td><td>10.14.6</td></tr>
<table>

**Comment by JCash** [comment]
Hmm, I cannot reproduce this. On my machine the editor stays at ~6-7% when I have the engine running on my machine (Mojave 10.14.5). I do have a dedicated gfx card on my iMac though, so I'll test on my MBP later.

**Comment by tadaaagames** [comment]
Yeah, I suspect it has something to do with the integrated gfx. Need to check if it also happens just on the notebook monitor
